### PR TITLE
Seed database

### PR DESCRIPTION
**Pull request by caithomas6789**
- The continent , botanist, origin and plant tables have been seeded.
- Might add a new country table to avoid repeating countries in the origin table.
- Left placeholder rows for the two missing plants. The plants are plant ID 7 and 43.

### CODE REVIEW

**Review by Abbey-Wilkinson — COMMENTED** [review]
Is epsilon.origin supposed to be s_epsilon because that is what the schema is called?

File: schema.sql [hunk 1]
```diff
@@ -1,54 +1,219 @@
...
+GO
+DROP TABLE IF EXISTS s_epsilon.botanist;
 GO
-DROP TABLE IF EXISTS botanist;
+DROP TABLE IF EXISTS s_epsilon.origin;
 GO
-DROP TABLE IF EXISTS origin;
+DROP TABLE IF EXISTS s_epsilon.country;
+GO
+DROP TABLE IF EXISTS s_epsilon.continent;
```
Comment (OscarB28) [inline schema.sql:14]: good to do this to make sure you're getting fresh tables

File: schema.sql [hunk 2]
```diff
@@ -1,54 +1,219 @@
...
-    last_watered TIMESTAMPTZ NOT NULL,
     plant_id INT NOT NULL,
-    PRIMARY KEY (plant_condition_id),
     FOREIGN KEY (plant_id)
-        REFERENCES plant(plant_id)
+        REFERENCES s_epsilon.plant(plant_id)
 );
 
+GO
+
```
Comment (OscarB28) [inline schema.sql:71]: the amount of seeding is impressive

File: schema.sql [hunk 3]
```diff
@@ -1,54 +1,219 @@
...
+        REFERENCES s_epsilon.country(country_id)
 );
 
-CREATE TABLE botanist (
-    botanist_id INT GENERATED ALWAYS AS IDENTITY,
+CREATE TABLE s_epsilon.botanist (
+    botanist_id INT NOT NULL IDENTITY(1,1) PRIMARY KEY,
     first_name VARCHAR(25) NOT NULL,
     surname VARCHAR(25) NOT NULL,
     email VARCHAR(30) UNIQUE NOT NULL,
```
Comment (OscarB28) [inline schema.sql:44]: possible that an email could be longer than this

**Review by OscarB28 — APPROVED** [review]
All looks good, I would only say that the VARCHAR(n) amounts look random

**Pull request merged**